scheds/rust/scx_lavd/src/bpf/main.bpf.c

```diff
@@ -362,6 +362,14 @@ static struct task_ctx *get_task_ctx(struct task_struct *p)
 	return taskc;
 }
 
+static s32 get_task_cpu_id(struct task_struct *p)
```

Comment:
You should be able to use `scx_bpf_task_cpu()` instead.

Reply:
Ah, got it. I will change it.